spec/dor/publishable_spec.rb

```diff
@@ -180,11 +181,12 @@ class ItemizableItem < ActiveFedora::Base
         expect(releases.map{ |r| r['to']}).to eq ['Searchworks', 'Some_special_place']
       end
 
-      it 'does include a releaseData element when there is content inside it' do
+      it 'include a releaseData element when there is content inside it, but does include this release data in identityMetadata' do
```

Comment:
Shouldn't this comment be "but does not include this release data in identityMetadata"?